### PR TITLE
Dummy

### PR DESCRIPTION
**Pull request by KorpelainenTommi**
Dummy responds to post requests with example json in a format similar to OpenAI

### CODE REVIEW

File: dummy/src/services/generateData.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import fs from 'fs';
+import { AIResponse } from '../types/AIResponse';
+
+const exampleData: AIResponse[] = JSON.parse(
+    fs.readFileSync('example_data.json').toString()
+) as AIResponse[];
+
+const generateData = (): AIResponse => {
+    const time = new Date().valueOf();
```
Comment (itulkki) [inline dummy/src/services/generateData.ts]: Date.now() could be used instead

File: dummy/src/services/generateData.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    const data = exampleData[Math.floor(Math.random() * exampleData.length)];
+    data.created = time;
+    return data;
```
Comment (itulkki) [inline dummy/src/services/generateData.ts:12]: We might want to add the prompt string and current time in a human readable format to the returned data to make dubugging easier.

**Review by itulkki — APPROVED** [review]
The data returned by dummy/src/services/generateData.ts could include the original prompt and generation date to make debugging easier. However, this is not critical and can be done later.

File: dummy/src/services/generateData.ts (hunk 1 above)
Comment (KorpelainenTommi) [reply dummy/src/services/generateData.ts]: Implemented 👍

File: dummy/src/services/generateData.ts (hunk 2 above)
Comment (KorpelainenTommi) [reply dummy/src/services/generateData.ts:12]: Added new type: `DummyResponse`
This contains both the GPT3 format response, as well as additional data in the form of `DebugInfo`. It contains a timestamp as a UTC date, as well as the given prompt (including text and all other fields)

**Pull request merged**